app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.app">
-
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <!-- TODO(#56): Reenable landscape support. -->
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
We should keep alphabetical order in this too.

Reply:
Done